clang/lib/Headers/adxintrin.h

```diff
@@ -14,9 +14,6 @@
 #ifndef __ADXINTRIN_H
 #define __ADXINTRIN_H
 
-/* Define the default attributes for the functions in this file. */
-#define __DEFAULT_FN_ATTRS __attribute__((__always_inline__, __nodebug__))
```

Comment:
(minor) You could put this back with `__attribute__((__always_inline__, __nodebug__, __target__("adx")))` and now use it for the ADX intrinsics.